tests/native/Cargo.toml

```diff
@@ -3,10 +3,14 @@ name = "tests"
 version = "0.1.0"
 authors = ["The Neon Community"]
 license = "MIT/Apache-2.0"
+build = "build.rs"
 
 [lib]
 name = "tests"
 crate-type = ["dylib"]
 
+[dev-dependencies]
+neon-build = "0.1.11"
+
```

Comment:
Also filed neon-bindings/neon-cli#46 and committed da0160757b233f1fcd9389041be61f74245076b4 to fix the template in the CLI.